releasenotes/notes/new-namespace-9c3b9fd73ed504e6.yaml

```
@@ -0,0 +1,15 @@
+---
+upgrade:
+  - |
+    The ``qiskit-aer`` Python package has moved to be a self contained
+    namespace, ``qiskit_aer``. Previously, the ``qiskit-aer`` package shared
+    a namespace with ``qiskit-terra``, ``qiskit.providers.aer``, however this
+    was problematic for several reasons (refer to
+    https://github.com/Qiskit/qiskit/issues/559 for more details) and this
+    release moved away from it. For the time being the ``qiskit.providers.aer``
+    import will continue to work and imports and usage of it will be redirected
+    to ``qiskit_aer`` automatically. The imports from the legacy
```

```suggestion
    The ``qiskit-aer`` Python package has moved to be a self-contained
    namespace, ``qiskit_aer``. Previously, it shared
    a namespace with ``qiskit-terra`` by being ``qiskit.providers.aer``.
    `This was problematic for several reasons <https://github.com/Qiskit/qiskit/issues/559>`__,
    and this release moves away from it. For the time being ``import qiskit.providers.aer``
    will continue to work and redirect to ``qiskit_aer`` automatically. Imports from the legacy
```

(just removing a bit of repetition to make it easier to read)